binchihua/src/orch/orch.c

```diff
@@ -1,16 +1,20 @@
+#include <stdio.h>
+#include <stdlib.h>
+
 #include "../../../libchihua-msg/include/orchestrator.h"
+#include "../common/common.h"
 #include "../ini/config.h"
 #include "opt.h"
 
-#include <stdio.h>
-#include <stdlib.h>
-
 /* A function that takes care of printing the lines of the ini file  */
 // NOLINTNEXTLINE
+#if 0
```

Comment:
This function was a first prototype for the ini parsing from Veronika and is not used anymore. Please remove it.

Reply:
Will do.